src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IDynamicInvocationExpression.cs

```diff
@@ -0,0 +1,448 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void DynamicInvocation_DynamicArgument()
+        {
+            string source = @"
+class C
+{
+    void M(C c, dynamic d)
+    {
+        /*<bind>*/c.M2(d)/*</bind>*/;
+    }
+
+    public void M2(int i)
+    {
+    }
+}
+";
+            string expectedOperationTree = @"
+IDynamicInvocationExpression (OperationKind.DynamicInvocationExpression, Type: dynamic) (Syntax: 'c.M2(d)')
+  Expression: IOperation:  (OperationKind.None) (Syntax: 'c.M2')
```

Comment:
>OperationKind.None [](start = 28, length = 18)

This is unexpected. As far as I understand we do not plan to expose method groups. Should we create DynamicMemberAccess node?

Reply:
@heejaechang Is going to (already done?) work on a change that will expose children of bound method and property group.